0000-ledger-plugin-interface-2/0000-ledger-plugin-interface-2.md

```diff
@@ -22,9 +22,12 @@ This spec depends on the [ILP spec](../0003-interledger-protocol/).
 | | [**connect**](#ledgerpluginconnect) ( options ) `⇒ Promise.<null>` |
 | | [**disconnect**](#ledgerplugindisconnect) ( ) `⇒ Promise.<null>` |
 | | [**isConnected**](#ledgerpluginisconnected) ( ) `⇒ Boolean` |
-| | [**sendTransfer**](#ledgerpluginsendtransfer) ( transfer ) <code>⇒ Promise.&lt;[FulfillmentInfo](#class-fulfillmentinfo)></code> |
-| | [**registerTransferHandler**](#ledgerpluginregistertransferhandler) ( transferHandler ) <code>⇒ null</code> |
-| | [**deregisterTransferHandler**](#ledgerpluginderegistertransferhandler) ( ) <code>⇒ null</code> |
+| | [**sendData**](#ledgerpluginsenddata) ( data ) <code>⇒ Promise.&lt;Buffer></code> |
+| | [**sendMoney**](#ledgerpluginsendmoney) ( amount ) <code>⇒ Promise.&lt;Buffer></code> |
```

Comment:
Nitpick: `sendMoney` or `sendValue`?

Reply:
sendValue would be a confusing name, because it sounds like it takes a value and communicates that value to your counterparty. sendMoney is technically not correct in all circumstances, but I think it makes more sense

Reply:
sendAsset?